Added help, added padding-right on label

grommas opened the pull request.

johannschopplich commented:
Thanks, well done contribution! I haven't had the time yet to take a look, hence my late reply.

In general: I'd prefer a `display: flex;` solution instead of the `<table>` tag for better responsiveness. I think with`flex-grow` and `flex-shrink` we don't even have to add media queries. Would you mind updating the PR?

grommas commented:
Hi Johann

I changed the code of the previous pull request to work with Kirby's native `k-column`. Compiled the JS and CSS files and updated the versions with a minor increment from 2.0.1 to 2.1.0. Hope that’s fine.

All the best from Bremen
Jakob

johannschopplich commented:
Hi Jakob, 

sorry for the delay. Great idea to use the native `<k-column>` tag. I couldn't add changes to your PR, that's why I merged it and did some styling simplifications. Afterwards, v2.1.0 was released.

Greetz from Leipzig to Bremen. I like the Bremer Filmkunst Theater very much.
